frontend/src/pages/articles/articles-feed.tsx

```diff
@@ -1,12 +1,36 @@
+import {
+  useAppDispatch,
+  useAppSelector,
+  useEffect,
+} from '~/libs/hooks/hooks.js';
+import { type UserDetailsResponseDto } from '~/packages/users/users.js';
+import { actions as articlesActions } from '~/slices/articles/articles.js';
+
 import { ArticleCard } from './components/components.js';
-import { MOCKED_ARTICLES, MOCKED_USER } from './libs/constants.js';
-
-const ArticlesFeed: React.FC = () => (
-  <>
-    {MOCKED_ARTICLES.map((article) => (
-      <ArticleCard key={article.id} article={article} user={MOCKED_USER} />
-    ))}
-  </>
-);
+import { MOCKED_REACTIONS, MOCKED_TAGS } from './libs/constants.js';
+
+const ArticlesFeed: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const { articles } = useAppSelector(({ articles }) => articles);
+
+  useEffect(() => {
+    void dispatch(articlesActions.fetchAll());
+  }, [dispatch]);
+
+  return (
+    <>
+      {articles.length > 0 &&
```

Comment:
```suggestion
      {Boolean(articles.length) &&
```

Reply:
It also throws an error
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/84803938/a6a98b39-0bf1-4ac7-838f-28536ac64a45)

Reply:
Can I change configuration or leave this part and the part with null assertion above as is?

Reply:
Yep,
Change eslint config for this rule please